README.md

```diff
@@ -5,6 +5,13 @@ Node.js API (N-API), along with library code that enables
 backward-compatibility with use with older versions of Node.js that do
 not have N-API built-in.
 
+### API Documentation
+
+ - [ABI-Stable C APIs in Node.js](https://nodejs.org/dist/latest-v7.x/docs/api/n-api.html)
```

Comment:
`latest-v7.x` only works by accident (and it might/should get deleted) – I would just stick with linking to https://nodejs.org/api/n-api.html and accept that it only works once the 8.0.0 release is out